modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```
@@ -10568,6 +10569,9 @@ object-action-trigger[onAfterUpdate]=On After Update
 object-configuration-name=Object
 object-definition-data=Object Data
 object-definition-id=Object ID
+object-definition-salesforce=Salesforce
```

Why not use the key as `salesforce` only?
```suggestion
salesforce=Salesforce
```